src/python/bot/fuzzers/syzkaller/config.py

```diff
@@ -48,7 +48,7 @@ def generate(serial,
   data['sandbox'] = 'android'
   data['ignores'] = ['WARNING:', 'INFO:']
   data['type'] = 'adb'
-  data['procs'] = 1
+  data['procs'] = 2
```

Comment:
The original config code did:
// By default, use half of the available CPUs
        if (mProcs <= 0) {
            mProcs = getNumProcs(device) / 2;

On a pixel 2 this is 8 cores/2 = 4 procs.  I would suggest using 4, 2 will fuzz too slowly, 8 will probably cause the device to lock up

            // Ensure we use at least 1 CPU
            mProcs = Integer.max(1, mProcs);
        }

Reply:
Done.